data/data_replication/json_cleaning_ACM.py

```diff
@@ -0,0 +1,78 @@
+import json
+from nameparser import HumanName
+
+with open('ACM.json') as f:
+  data = json.load(f)
+
+# for each item, delete unnecessary author info
+for i in range(205):
```

Comment:
@csav3 , can you explain the `for i in range(205)` part? Are you allowing up to 205 exceptions? How did you come to this number?

Reply:
That's the number of items in the dataset. I'll change this by storing the number in a descriptive variable. Sorry for not making that clear!